osquery/requests/serializers/json_tests.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include "osquery/requests/serializers/json.h"
+
+namespace osquery {
+
+class JSONTests : public testing::Test {};
+
+TEST_F(JSONTests, testSerialize) {
```

Comment:
Eh, isn't this just testing boost's JSON parsing? If it's placed somewhere more general like core/text then absolutely.